src/controller/CHIPDevice.cpp

```diff
@@ -76,7 +76,7 @@ CHIP_ERROR Device::SendMessage(Protocols::Id protocolId, uint8_t msgType, System
     if (!loadedSecureSession)
     {
         // Secure connection already existed
-        // Hold on to the buffer, in case session resumption and resend is needed
+        // Hold on to the buffer, in case session resumption and resen is needed
```

Comment:
I think this was a mistake and should remain "resend"

Reply:
Good catch, yes.